.github/workflows/ci.yaml

```diff
@@ -34,6 +34,9 @@ jobs:
             spog/ui/target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v2
```

Comment:
what's this for?

Reply:
Per additional commit, it's due to guac-rs having to compile a .proto for interacting with GUAC over gRPC.